src/utils/testResult.js

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line import/prefer-default-export
+export const testResult = (message, pass) => ({
+  message: () => message,
+  pass,
+});
```

Comment:
If you're happy with this refactor I'll roll this into the other assertions in the next PR